### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by KyleSaunders1310**

### CODE REVIEW

File: src/app/components/navbar/navbar.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div id="navbar" class="flex-container">
+    <div id="navbar-title" class="flex-container">
```
Comment (Solenoden) [inline src/app/components/navbar/navbar.component.html:2]: Are there no styles linked to this css class? If not please remove it

File: src/app/components/navbar/navbar.component.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#navbar-item:hover{
+    transform: perspective(1px) scale(1.1);
+    transition: all .5s ease;
+}
+
+
+div.isSelected{
+    border-bottom: 3px solid material.get-color-from-palette($pink-palette, 500) !important;
+}
+p.isSelected{
```
Comment (Solenoden) [inline src/app/components/navbar/navbar.component.scss:74]: Please reformat to the css standard of kebab-case

File: src/app/components/navbar/navbar.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <img src="../../../assets/images/TitleIcon.png">
+        <h1 class="paragraph-header accent">KYLE<br>SAUNDERS</h1>
+    </div>
+    <div id="navbar-items" class="flex-container">
+        <div id="navbar-item" class="isSelected">
+            <p class="paragraph-subheader isSelected">About me</p>
+        </div>
+        <div id="navbar-item">
```
Comment (Solenoden) [inline src/app/components/navbar/navbar.component.html:10]: Change to css class

**Review by Solenoden — CHANGES_REQUESTED** [review]

File: src/app/components/navbar/navbar.component.html (hunk 1 above)
Comment (KyleSaunders1310) [reply src/app/components/navbar/navbar.component.html:2]: I have removed the class.

File: src/app/components/navbar/navbar.component.scss (hunk 2 above)
Comment (KyleSaunders1310) [reply src/app/components/navbar/navbar.component.scss:74]: Fixed it.

File: src/app/components/navbar/navbar.component.html (hunk 3 above)
Comment (KyleSaunders1310) [reply src/app/components/navbar/navbar.component.html:10]: Fixed it.

**Review by Solenoden — APPROVED** [review]

**Pull request merged**